.github/workflows/ci.yaml

```diff
@@ -7,64 +7,120 @@ on:
     branches: [main]
 
 jobs:
-  build:
-    services:
-      localstack:
-        # Provides local ephemeral versions of AWS services to use during tests.
```

Comment:
I've removed this for now. After talking with Phil, I confirmed that our S3 connector tests are hitting the real S3 endpoints and this wasn't being used at all.